invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/index.js

```diff
@@ -6,19 +6,28 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import _truncate from "lodash/truncate";
 import { createSearchAppInit } from "@js/invenio_search_ui";
 
-import _upperFirst from "lodash/upperFirst";
 import { ManagerMembersResultsItem } from "./ManagerMembersResultItem";
 import { MembersSearchBarElement } from "../../components/MembersSearchBarElement";
 import { MembersResults } from "../components/MembersResult";
 import { MembersResultsGridItem } from "../components/MembersResultsGridItem";
 import { MembersSearchLayout } from "../components/MembersSearchLayout";
 import { ManagerMembersResultsContainer } from "./ManagerMembersResultContainer";
+import { parametrize } from 'react-overridable';
+
+
+const domContainer = document.getElementById("community-members-search-root");
+const communitiesRoles = JSON.parse(domContainer.dataset.communitiesRoles);
```

Comment:
I think we will have to do the same for the visibility?

Reply:
visibility is still not implemented